src/components/ArtistName.jsx

```diff
@@ -0,0 +1,20 @@
+import './ArtistName.css'; 
+
+export const ArtistName = ({artistName,artistLink,index,length})=>{
+    const clickArtistName = ()=>{
+        window.location.href=artistLink
+    }
+    const commaArtistName = (index,length) => {
+        if (index === length - 2) {
+            return ' &'
+        } else if (index < length -2) {
+            return ' ,'
+        } else{
+            return ' '

```

Comment:
Nice solution! Can you think of a way to not include the & in the first link 👀 

![Skärmavbild 2024-03-13 kl  15 06 25](https://github.com/Technigo/project-music-releases-vite/assets/51529201/6bf111b3-b2b1-4b9a-991a-dcad519ef42d)

Reply:
I solved this, but I don't know why it became 
![Screenshot 2024-03-15 at 23 51 25](https://github.com/Technigo/project-music-releases-vite/assets/140561745/db9c6796-3ef5-4a30-8f3e-5cd3fab80554)
,  I want there is also a space after &. I added a space after & in this code `return '  &  '`, but it doesn't work.